<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreen</class>
 <widget class="QWidget" name="GameScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1047</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Contenedor" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1080</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QWidget" name="Mesa" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#Mesa{
	border-image: url(:/Resources/mesa2.png);
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="Nose" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1350</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#Nose{
	border-image: url(:/Resources/fondo.png);
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QWidget" name="FondoNPC" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#FondoNPC{
	border-image: url(:/Resources/fondoAlturas.png);
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QWidget" name="Escritorio" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1350</width>
          <height>1080</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#Escritorio{
	border-image: url(:/Resources/escritorio2.png);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="ContenedorBotones" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1920</width>
             <height>150</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="BotonCentrar">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>75</height>
               </size>
              </property>
              <property name="text">
               <string>Centrar documentos</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="aceptarBoton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>75</height>
               </size>
              </property>
              <property name="text">
               <string>Aceptar</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rechazarBoton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>75</height>
               </size>
              </property>
              <property name="text">
               <string>Rechazar</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="Separador2" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1920</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="Separador1" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1920</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
